test/system/211-dbus.bats

```diff
@@ -87,3 +87,311 @@ teardown() {
+}
+
+@test "dbus: session bus inside Ubuntu 18.04" {
+  create_distro_container ubuntu 18.04 ubuntu-toolbox-18.04
+
+  expected_response="$(busctl call \
+                        --user org.freedesktop.DBus \
+                        /org/freedesktop/DBus \
+                        org.freedesktop.DBus.Peer Ping)"
+
+  run --keep-empty-lines --separate-stderr "$TOOLBOX" run --distro ubuntu --release 18.04 sh -c 'busctl call \
+                                                                                                --user org.freedesktop.DBus \
+                                                                                                /org/freedesktop/DBus org.freedesktop.DBus.Peer Ping'
+
+  assert_success
+  assert_equal "${lines[*]}" "$expected_response" 
+
+  # shellcheck disable=SC2154
+  assert [ ${#stderr_lines[@]} -eq 0 ]
+}
+
+@test "dbus: system bus inside Ubuntu 18.04" {
+  create_distro_container ubuntu 18.04 ubuntu-toolbox-18.04
+
+  expected_response="$(busctl call \
+                        --system org.freedesktop.systemd1 \
+                        /org/freedesktop/systemd1 \
+                        org.freedesktop.DBus.Properties Get "ss" \
+                        org.freedesktop.systemd1.Manager Version)"
+
+  run --keep-empty-lines --separate-stderr "$TOOLBOX" run --distro ubuntu --release 18.04 sh -c  'busctl call \
+                                                                          --system org.freedesktop.systemd1 \
+                                                                          /org/freedesktop/systemd1 \
+                                                                          org.freedesktop.DBus.Properties Get "ss" \
+                                                                          org.freedesktop.systemd1.Manager Version'
+
+  assert_success
+  assert_line --index 0 "$expected_response"
+
+  if check_bats_version 1.10.0; then
+    assert [ ${#lines[@]} -eq 1 ]
+  else
+    assert [ ${#lines[@]} -eq 2 ]
+  fi
+
+  # shellcheck disable=SC2154
+  assert [ ${#stderr_lines[@]} -eq 0 ]
+}
+
+@test "dbus: session bus inside Ubuntu 20.04" {
+  create_distro_container ubuntu 20.04 ubuntu-toolbox-20.04
+
+  expected_response="$(busctl call \
+                        --user org.freedesktop.DBus \
+                        /org/freedesktop/DBus \
+                        org.freedesktop.DBus.Peer Ping)"
+
+  run --keep-empty-lines --separate-stderr "$TOOLBOX" run --distro ubuntu --release 20.04 sh -c 'busctl call \
+                                                                                                --user org.freedesktop.DBus \
+                                                                                                /org/freedesktop/DBus \
+                                                                                                org.freedesktop.DBus.Peer Ping'
+
+  assert_success
+  assert_equal "${lines[*]}" "$expected_response" 
```

Comment:
Nitpick: there's a trailing whitespace here.

You can use `git config --global color.diff auto` to enable coloured output from `git diff`, `git show`, etc..  It will make it easier to spot these.

Reply:
I took the liberty to remove these.